src/core/server/deprecations/deprecations_service.ts

```diff
@@ -37,49 +37,7 @@ import { SavedObjectsClientContract } from '../saved_objects/types';
  *
  * async function getDeprecations({ esClient, savedObjectsClient }: GetDeprecationsContext): Promise<DeprecationsDetails[]> {
  *   const deprecations: DeprecationsDetails[] = [];
- *   const count = await getTimelionSheetsCount(savedObjectsClient);
```

Comment:
I don't think this example should be removed. Or provide an updated version, please

Reply:
Can someone provide example which I should put here? Because this deprication will invalid after merge timelion app removal.

Reply:
It's an example in deprecation_service doc, so it can any fictitious example of code. Renaming `timelion` to `foo` might do the trick.

Reply:
done